knowledge-base/dropdowns-disable-long-text-wrap.md

```diff
@@ -0,0 +1,113 @@
+---
+title: DropDown Items Wrap or Overlap
+description: Learn how to resolve and prevent text wrapping of long items in Telerik Blazor dropdown components, such as AutoComplete, ComboBox, DropDownList, and MultiSelect.
+type: troubleshooting
+page_title: How to Prevent Dropdown Item Text Wrapping
+slug: dropdowns-kb-disable-long-text-wrap
+tags: blazor, autocomplete, combobox, dropdownlist, multiselect, css, styling
+ticketid: 1663555, 1668373, 1679799, 1682531
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                AutoComplete for Blazor, <br />
+                ComboBox for Blazor, <br />
+                DropDownList for Blazor, <br />
+                MultiSelect for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article deals with the following scenarios:
+
+1. The items in a Telerik dropdown component are long and wrap to multiple lines. The items become very high. How to avoid and disable this behavior?
```

Comment:
These should not be an ordered list, since they are two separate scenarios. I recommend going with simple bullets instead (unordered list).